src/script/view_model/ContentViewModel.ts

```diff
@@ -178,23 +176,13 @@ export class ContentViewModel {
       repositories.user,
       repositories.message,
     );
-    this.titleBar = new TitleBarViewModel(mainViewModel.calling, mainViewModel.panel, this, repositories.calling);
 
     this.historyExport = new HistoryExportViewModel(repositories.backup);
     this.historyImport = new HistoryImportViewModel(repositories.backup);
 
     this.state.subscribe(state => {
-      switch (state) {
-        case ContentViewModel.STATE.CONVERSATION:
-          this.inputBar.addedToView();
```

Comment:
I believe we need to keep the `this.inputBar.addedToView()` this has not been migrated yet

Reply:
Yes, good catch 👼  I read it as titleBar 🤦